### PR TITLE
fix: point ecommerce/ecommerce-worker giturls to openedx github org

### PR DESCRIPTION
**Pull request by christopappas**
2U is not currently forking ecommerce/ecommerce-worker back to the edX github org, but I wanted to make this change now as to not forget later should the a fork be made.

### CODE REVIEW

File: changelog.d/20240417_144650_cpappas_update_more_ecommerce_git_urls.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<!--
+Create a changelog entry for every new user-facing change. Please respect the following instructions:
+- Indicate breaking changes by prepending an explosion 💥 character.
+- Prefix your changes with either [Bugfix], [Improvement], [Feature], [Security], [Deprecation].
+- You may optionally append "(by @<author>)" at the end of the line, where "<author>" is either one (just one)
+of your GitHub username, real name or affiliated organization. These affiliations will be displayed in
+the release notes for every release.
+-->
+
+<!-- - 💥[Feature] Foobarize the blorginator. This breaks plugins by renaming the `FOO_DO` filter to `BAR_DO`. (by @regisb) -->
```
Comment (DawoudSheraz) [inline changelog.d/20240417_144650_cpappas_update_more_ecommerce_git_urls.md]: nit: Please remove these comments.

**Review by DawoudSheraz — APPROVED** [review]

File: changelog.d/20240417_144650_cpappas_update_more_ecommerce_git_urls.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+[Improvement] - Change ecommerce and ecommerce-worker remotes from `edx` to `openedx`. (by @christopappas)
```
Comment (Faraz32123) [inline changelog.d/20240417_144650_cpappas_update_more_ecommerce_git_urls.md]: - [Improvement] Change ecommerce and ecommerce-worker remotes from `edx` to `openedx`. (by @christopappas)

File: changelog.d/20240417_144650_cpappas_update_more_ecommerce_git_urls.md (hunk 1 above)
Comment (christopappas) [reply changelog.d/20240417_144650_cpappas_update_more_ecommerce_git_urls.md]: Done

File: changelog.d/20240417_144650_cpappas_update_more_ecommerce_git_urls.md (hunk 2 above)
Comment (christopappas) [reply changelog.d/20240417_144650_cpappas_update_more_ecommerce_git_urls.md]: I have removed the "-" following [Improvement]

Comment (Faraz32123) [reply changelog.d/20240417_144650_cpappas_update_more_ecommerce_git_urls.md]: "-" will come before [Improvement].

Comment (christopappas) [reply changelog.d/20240417_144650_cpappas_update_more_ecommerce_git_urls.md]: updated 👍

**Review by Faraz32123 — APPROVED** [review]
LGTM!

**Pull request merged**